pom.xml

```diff
@@ -118,7 +118,7 @@
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-    <tag>HEAD</tag>
+    <tag>warnings-4.61</tag>
```

Comment:
Did you check if the [ATH test](https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/test/java/plugins/WarningsPluginTest.java) still is running after the UI changes?